### PR TITLE
Use Embed for !help command

### PR DESCRIPTION
**Pull request by olliy1or**
Embed for help command

### CODE REVIEW

**Comment by LoneWolfHT** [comment]
image

**Comment by olliy1or** [comment]
looks good to me

File: index.js [hunk 1]
```diff
@@ -186,7 +186,16 @@ client.on("message", message => {
 			message.channel.send(formatLeaderboard(orderedList));
 		}
 	} else if (command == "help") {
-		message.channel.send("Commands: `!rank`, `!oldrank`, `!oldleaders`, `!help`");
+		const helpEmbed = new Discord.MessageEmbed()
+        	.setTitle("Commands")
```
Comment (rubenwardy) [inline index.js]: mixed in dentation here - should be tabs

Comment (LoneWolfHT) [reply index.js]: Fixed now unless Github messed with it

File: index.js [hunk 2]
```diff
@@ -186,7 +186,16 @@ client.on("message", message => {
 			message.channel.send(formatLeaderboard(orderedList));
 		}
 	} else if (command == "help") {
-		message.channel.send("Commands: `!rank`, `!oldrank`, `!oldleaders`, `!help`");
+		const helpEmbed = new Discord.MessageEmbed()
+			.setTitle("Commands")
+			.setColor("#0000E5")
+			.addField(prefix + "rank", "Shows ingame rankings", false)
+			.addField(prefix + "oldrank", "Shows old rankings", false)
+			.addField(prefix + "leaders", "Shows the top 60 leaderboard or links you to the channel for it", false)
```
Comment (rubenwardy) [inline index.js]:
```suggestion
			.addField(prefix + "leaders", "Shows the top 60 leaderboard or links to the dedicated channel for it", false)
```

**Review by rubenwardy — APPROVED** [review]

**Comment by rubenwardy** [comment]
Are you ready for it to be merged?

**Comment by olliy1or** [comment]
> Are you ready for it to be merged?

yeah, doesnt look like it has anything else needed added or removed

**Pull request merged**